sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -548,34 +552,59 @@ public virtual Response ActivateMultiSlot(string eventId, CancellationToken canc
             }
         }
 
-        /// <summary> Gets the configuration details for the live model to use </summary>
-        internal Configuration GetConfigurationForLiveModel(string authType, string authValue)
+        /// <summary> Gets the rank process initiated with live model to use </summary>
+        internal RankProcessor GetConfigurationForLiveModel(string authType)
         {
-            _personalizerServiceProperties = ServiceConfigurationRestClient.Get();
-            _personalizerPolicy = PolicyRestClient.Get();
+            liveModelLastRefresh = DateTimeOffset.UtcNow;
             Configuration config = new Configuration();
             // set up the model
             if (authType == "apiKey")
             {
-                config["http.api.key"] = authValue;
+                config["http.api.key"] = _azureKeyCredential.Key;
             }
             else
             {
-                //ToDo: TASK 13057958 Working on changes to support token authentication in RLClient
-                //config["http.token.key"] = authValue;
+                var tokenRequestContext = new TokenRequestContext(scopes);
+                CancellationToken cancellationToken = default;
+                AccessToken token = _tokenCredential.GetToken(tokenRequestContext, cancellationToken);
+                config["http.api.key"] = "Bearer " + token.Token;
+                config["HTTP_API_HEADER_KEY_NAME"] = "Authorization";
```

Comment:
I must have missed this in the VW PR. Please fix this naming to be consistent.

"http.api.key" and "HTTP_API_HEADER_KEY_NAME" are not in alignment at all. They have different delimiters, case, and structure. Please make them close to the same.